views/marts/product/fct_active_users.view.lkml

```diff
@@ -76,31 +76,86 @@ view: fct_active_users {
   dimension: daily_desktop_active_users {
     type: number
     sql: ${TABLE}.daily_desktop_active_users ;;
-    label: "Desktop/Client DAU"
-    description: "DAU reported by Rudderstack telemetry."
+    label: "Desktop DAU"
+    description: "Desktop DAU reported by telemetry."
```

Comment:
It's worth mentioning that this is coming from client. The same convention as in DAU can be used for label and description.

Reply:
This is both Client (Rudderstack) and Legacy (Segment). I updated it to reflect the same.